From 56da7f5deed2697e63c2873a4e20b35ae6f1e546 Mon Sep 17 00:00:00 2001
From: Paul Gortmaker <paul.gortmaker@windriver.com>
Date: Wed, 7 Jul 2021 20:55:53 -0400
Subject: [PATCH] xfrm: fix up warning introduced by pseudo revert

To fix:
  net/xfrm/xfrm_state.c:47:42: warning: 'xfrm_state_hash_generation' defined but not used

The line was removed in the 5.10-stable backport of commit e88add19f681
("net: xfrm: Localize sequence counter per network namespace") but then
was incorrectly reintroduced in the commit referenced below.

Fixes: 71b740f9373d ('Revert "net/xfrm: fixup 5.10.30 -stable merge"')

Upstream-Status: Pending

Cc: Yanfei Xu <yanfei.xu@windriver.com>
Signed-off-by: Paul Gortmaker <paul.gortmaker@windriver.com>
Signed-off-by: Bruce Ashfield <bruce.ashfield@gmail.com>
---
 net/xfrm/xfrm_state.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/net/xfrm/xfrm_state.c b/net/xfrm/xfrm_state.c
index e0152019db6f..4c7cae69ea29 100644
--- a/net/xfrm/xfrm_state.c
+++ b/net/xfrm/xfrm_state.c
@@ -44,7 +44,6 @@ static void xfrm_state_gc_task(struct work_struct *work);
  */
 
 static unsigned int xfrm_state_hashmax __read_mostly = 1 * 1024 * 1024;
-static __read_mostly seqcount_spinlock_t xfrm_state_hash_generation;
 static struct kmem_cache *xfrm_state_cache __ro_after_init;
 
 static DECLARE_WORK(xfrm_state_gc_work, xfrm_state_gc_task);
-- 
2.27.0

